docs/document/content/manual/sharding-jdbc/configuration/config-java.cn.md

```diff
@@ -296,12 +296,12 @@ ShardingStrategyConfiguration的实现类，用于配置不分片的策略。
 | *名称*             | *数据类型*                    | *说明*                                                                         |
 | ----------------- | ---------------------------- | ------------------------------------------------------------------------------ |
 | column            | String                       | 自增列名称                                                                      |
-| type              | String                       | 自增列值生成器类型，可自定义或选择内置类型：SNOWFLAKE/UUID |
-| props             | Properties                   | 自增列值生成器的相关属性配置                                                      |
+| type              | String                       | 自增列值生成算法类型，可自定义或选择内置类型：SNOWFLAKE/UUID |
```

Comment:
KeyGeneratorConfigration contains KeyGenerateAlgorithm now, should move the modification to yaml and namespace config